docs/04. Seguimiento/4.8. Registro de cambios.mdx

```diff
@@ -24,29 +25,40 @@ planificación inicial.
+| 21 | Poder filtrar por fechas en las intervenciones | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 22 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 22 | Creación de familia, que se puedan añadir miembros | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 23 | Mostrar fecha de caducidad        | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 24 | Mostrar entregas                   | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 25 | Implementar recuperación de contraseña | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 26 | Mostrar mensajes de avisos en la aplicación | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 27 | Notificar cuando queda un tiempo determinado | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```

Comment:
Number 27 has been affected by a change in the scope of the project

Reply:
I updated the state to "cancelled"